.github/workflows/main.yml

```diff
@@ -23,15 +23,8 @@ jobs:
     # this decrypts the ssh key and stores it in $HOME/secrets/githubaction-key.pem
     # maybe we should add a step to delete after deploying?
     - name: Decrypt ssh key
-      run: ./.github/scripts/decrypt_secret.sh
-      env:
-        SSH_KEY_PASSPHRASE: ${{ secrets.SSH_KEY_PASSPHRASE }}
```

Comment:
pretty sure we still have to set this one

Reply:
done